vibi-dpu/src/github/auth.rs

```diff
@@ -1,3 +1,4 @@
+use chrono::DateTime;
```

Comment:
Is this intentional? 
I haven't seen the `chrono` crate being imported anywhere else.